mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -4,6 +4,7 @@ CLASS aij net/minecraft/entity/LivingEntity
 	FIELD aD lastLimbDistance F
 	FIELD aE limbDistance F
 	FIELD aF limbAngle F
+	FIELD aL previousYaw F
```

Comment:
prevYaw here, or previousHeadYaw in the other one

Reply:
Removed this mapping. This looks to me like some delay in yaw change instead... not sure.